packages/app/src/components/Bookmarks/BookmarkFolderMenuItem.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { useTranslation } from 'next-i18next';
+import {
+  DropdownItem,
+  DropdownMenu, DropdownToggle, UncontrolledDropdown,
+} from 'reactstrap';
+
+import { apiv3Post } from '~/client/util/apiv3-client';
+import { toastError, toastSuccess } from '~/client/util/toastr';
+import { BookmarkFolderItems } from '~/interfaces/bookmark-info';
+import { useSWRBookmarkInfo } from '~/stores/bookmark';
+import { useSWRxBookamrkFolderAndChild } from '~/stores/bookmark-folder';
+import { useSWRxCurrentPage } from '~/stores/page';
+
+import FolderIcon from '../Icons/FolderIcon';
+import TriangleIcon from '../Icons/TriangleIcon';
+
+import BookmarkFolderNameInput from './BookmarkFolderNameInput';
+
+
+type Props ={
+  item: BookmarkFolderItems
+  onSelectedChild: () => void
+  isSelected: boolean
+}
+const BookmarkFolderMenuItem = (props: Props):JSX.Element => {
+  const {
+    item, isSelected, onSelectedChild,
+  } = props;
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = useState(false);
+  const [currentChildFolders, setCurrentChildFolders] = useState<BookmarkFolderItems[]>();
+  const { data: childFolders, mutate: mutateChildFolders } = useSWRxBookamrkFolderAndChild(item._id);
+  const { mutate: mutateParentFolders } = useSWRxBookamrkFolderAndChild(item.parent);
+  const [selectedItem, setSelectedItem] = useState<string | null>(null);
+  const [isCreateAction, setIsCreateAction] = useState<boolean>(false);
+  const { data: currentPage } = useSWRxCurrentPage();
+  const { mutate: mutateBookmarkInfo } = useSWRBookmarkInfo(currentPage?._id);
+
+  const onPressEnterHandlerForCreate = useCallback(async(folderName: string) => {
+    try {
+      await apiv3Post('/bookmark-folder', { name: folderName, parent: item._id });
+      await mutateChildFolders();
+      setIsCreateAction(false);
+      toastSuccess(t('toaster.create_succeeded', { target: t('bookmark_folder.bookmark_folder') }));
```

Comment:
`toaster.create_succeeded` t function response seems not to display correctly. Please fix it.
I think it's because `toaster.create_succeeded` JSON dictionary is in the `commons.js`.